openpdf/pom.xml

```diff
@@ -54,7 +54,16 @@
             <version>${hamcrest.version}</version>
             <scope>test</scope>
         </dependency>
-
+        <dependency>
```

Comment:
Please set these dependecies as optional, and move them above of the <scope>test</scope>-Dependecies.

Reply:
I've updated dependencies.
I've verified output pdf with hindi language, following is the output:
Pre
![image](https://user-images.githubusercontent.com/6597998/73130925-efd77b00-4027-11ea-87da-a993e4270813.png)

Post
![image](https://user-images.githubusercontent.com/6597998/73130932-04b40e80-4028-11ea-97df-470fc1710cf2.png)

I'm facing couple of challenges with junit test, need your help.
1. Licensing of Font files for hindi or any other language. For ex https://docs.microsoft.com/en-us/typography/font-list/nirmala-ui
2. I wanted to verify PDF output in test case but seems PdfTextExtractor doesn't support characters other than english, please refer code snippet below. 

    `

            @Test
            public void testGlyphSubstitution() throws Exception{
                FontFactory.register("C:\\Windows\\Fonts\\NIRMALA.TTF");
                ByteArrayOutputStream fileBytes = new ByteArrayOutputStream();
                try (Document document = new Document();) {
                PdfWriter.getInstance(document, fileBytes);
                document.open();
                document.add(new Chunk("नमस्ते", FontFactory.getFont("nirmala ui", "Identity-H", false, 10, 0, null)));
                document.close();
                PdfReader pdfReader = new PdfReader(fileBytes.toByteArray());
                PdfTextExtractor pdfTextExtractor = new PdfTextExtractor(pdfReader);
                //Output is empty 
                System.out.println(pdfTextExtractor.getTextFromPage(0));
                System.out.println(pdfTextExtractor.getTextFromPage(1));

            }

`